src/components/capture/creation/DetailsForm.tsx

```diff
@@ -99,22 +101,7 @@ function NewCaptureDetails(props: NewCaptureDetailsProps) {
             </DialogContentText>
 
             <Stack direction="row" spacing={2}>
-                {error ? (
-                    error
-                ) : loading ? (
-                    <>
-                        <Skeleton
-                            variant="rectangular"
-                            height={40}
-                            width="50%"
-                        />
-                        <Skeleton
-                            variant="rectangular"
-                            height={40}
-                            width="50%"
-                        />
-                    </>
-                ) : connectors.length > 0 ? (
+                {isPostProcessingDone ? (
```

Comment:
When you have a post processor it does not matter if it is idle, loading, whatever... it shows loading until the processing is done.